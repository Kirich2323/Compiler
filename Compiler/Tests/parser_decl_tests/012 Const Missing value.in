const
    a = ;
    b = 0.5;