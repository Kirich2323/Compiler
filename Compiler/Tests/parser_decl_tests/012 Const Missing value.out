(2 ; 9): Invalid expression.